src/python/pants/backend/awslambda/python/rules_test.py

```diff
@@ -342,3 +342,75 @@ def test_layer_must_have_dependencies(rule_runner: PythonRuleRunner) -> None:
             expected_extra_log_lines=("    Runtime: python3.7",),
             layer=True,
         )
+
+
+def test_collisions_ok(rule_runner: PythonRuleRunner) -> None:
```

Comment:
Do we need these tests given the generic one in pex_venv_test.py ?

Reply:
I was assuming we want to validate we've hooked up the field appropriately.

I guess technically this could be a normal test that checks the `pex_venv.py` rule is called with the expected args, rather than needing to be a full integration test...

Do you think it's worth switching over?

Reply:
I've switched them all to be fully mocked, basically just testing that `PexCliProcess` gets the expected args.